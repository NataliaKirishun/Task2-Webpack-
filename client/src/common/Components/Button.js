import React from "react"
import styled from "styled-components"
import Theme from "../Theme/Theme"

const Button = styled.button`
    font-size: 16px;
    margin-left: auto;
    background-color: ${Theme.colors.red};
    border: 0;
    padding: 6px 12px;
    color: ${Theme.colors.white};
    text-transform: uppercase;
    border-radius: 4px;
    outline: none;
`;

export default Button;