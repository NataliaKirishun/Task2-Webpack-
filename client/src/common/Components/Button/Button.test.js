import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';

describe('Button Snapshot', () => {
    it('Button Common Component should render correctly with no props', () => {
        const component = shallow(<Button>Push</Button>);
        expect(component).toMatchSnapshot();
    });
});