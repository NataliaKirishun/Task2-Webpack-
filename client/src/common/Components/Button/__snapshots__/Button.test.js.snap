// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Snapshot Button Common Component should render correctly with no props 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    font-size: 16px;
    margin-left: auto;    
    border: 0;
    padding: 6px 12px;    
    text-transform: uppercase;
    border-radius: 4px;
    outline: none;
    cursor: pointer;
    
    ",
          [Function],
          "
    
    ",
          [Function],
          "
",
        ],
      },
      "displayName": "styled.button",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "button",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Push
</StyledComponent>
`;
